docs/resources/management_access_policy.md

```diff
@@ -0,0 +1,524 @@
+
+## Schema ##
+
+### Required ###
+
+* `name` (name) - (string) The name of the Management Access Policy object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the Management Access Policy object.
+
+### Optional ###
+
+* `annotation` (annotation) - (string) The annotation of the Management Access Policy object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the Management Access Policy object.
+* `name_alias` (nameAlias) - (string) The name alias of the Management Access Policy object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `strict_security_on_apic_oob_subnet` (strictSecurityOnApicOOBSubnet) - (string) Removes implicit access permissions from any APIC out-of-band (OOB) subnet IP address.
+  - Default: `false`
+  - Valid Values: `no`, `yes`.
+* `http_service` - (map) A map of Hypertext Transfer Protocol (HTTP) Service (ACI object [commHttp](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/commHttp/overview)).
+  #### Optional ####
+    
+    * `access_control_allow_credential` (accessControlAllowCredential) - (string) Determines if credentials are allowed in HTTP (Hypertext Transfer Protocol) responses.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+    * `access_control_allow_origins` (accessControlAllowOrigins) - (string) Determines which origins can access the resource. For example, inputting http://127.0.0.1:8000 would allow access from a source device with that IP address and port.
+    * `admin_state` (adminSt) - (string) The administrative state of the HTTP (Hypertext Transfer Protocol) Service.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+    * `annotation` (annotation) - (string) The annotation of the Hypertext Transfer Protocol (HTTP) Service object.
+      - Default: `orchestrator:terraform`
+    * `cli_only_mode` (cliOnlyMode) - (string) Determines if CLI-only mode is enabled.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+    * `description` (descr) - (string) The description of the Hypertext Transfer Protocol (HTTP) Service object.
+    * `global_throttle_rate` (globalThrottleRate) - (string) Maximum API calls allowed per unit time.
+      - Default: `10000`
+    * `global_throttle_state` (globalThrottleSt) - (string) The throttle state for all clients without specific identifier in their headers.
+      - Default: `disabled`
+      - Valid Values: `disabled`, `enabled`.
+    * `global_throttle_unit` (globalThrottleUnit) - (string) The unit of measurement for rate limiting.
```

Comment:
Based on the UI there only looks like two options.
1. Requests/Second
2. Requests/Minute

The metadata has an interesting regex validator `^[r]/[ms]$`
It can only really be `r/m` or `r/s`
Should we add these to valid values?

Reply:
we could also raise an issue (enhancement) for this to add regex validator when provided in meta as a validator function.